tosfs/core.py

```diff
@@ -413,6 +416,200 @@ def _try_dir_info(self, bucket: str, key: str, path: str, fullpath: str) -> dict
         except Exception as e:
             raise TosfsError(f"Tosfs failed with unknown error: {e}") from e
 
+    def exists(self, path: str, **kwargs: Union[str, bool, float, None]) -> bool:
+        """Check if a path exists in the TOS file system.
+
+        Parameters
+        ----------
+        path : str
+            The path to check for existence.
+        **kwargs : dict, optional
+            Additional arguments.
+
+        Returns
+        -------
+        bool
+            True if the path exists, False otherwise.
+
+        Raises
+        ------
+        tos.exceptions.TosClientError
+            If there is a client error while checking the path.
+        tos.exceptions.TosServerError
+            If there is a server error while checking the path.
+        TosfsError
+            If there is an unknown error while checking the path.
+
+        Examples
+        --------
+        >>> fs = TosFileSystem()
+        >>> fs.exists("tos://bucket/to/file")
+        True
+        >>> fs.exists("tos://mybucket/nonexistentfile")
+        False
+
+        """
+        if path in ["", "/"]:
+            # the root always exists, even if anon
```

Comment:
anon ? What's the meaning ?

Reply:
`anon` means `anonymous `, aligned with `s3fs`.